metrics/register/session_manager.go

```diff
@@ -0,0 +1,19 @@
+package register
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/rancher/norman/metrics"
+)
+
+func init() {
```

Comment:
Could this be moved to session_manager to avoid exporting all of these vars?

Reply:
Do you mean moving this logic to metrics/session_manager.go?